<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>342</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>342</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QListWidget" name="listWidget">
       <item>
        <property name="text">
         <string>Item 1</string>
        </property>
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Este é o item 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Item 2</string>
        </property>
        <property name="toolTip">
         <string>Este é o item 2</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit"/>
     </widget>
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string>Itens</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/radiobutton-off.png</normaloff>:/images/radiobutton-off.png</iconset>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valores</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/radiobutton.png</normaloff>:/images/radiobutton.png</iconset>
       </property>
      </column>
      <item>
       <property name="text">
        <string>Item1</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
       </property>
       <property name="text">
        <string/>
       </property>
       <item>
        <property name="text">
         <string>SubItem 11</string>
        </property>
        <property name="text">
         <string>Nulo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SubItem 12</string>
        </property>
        <property name="text">
         <string>Inválido</string>
        </property>
        <item>
         <property name="text">
          <string>SubItem 121</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SubItem 122</string>
         </property>
        </item>
       </item>
      </item>
      <item>
       <property name="text">
        <string>Item2</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
       </property>
       <item>
        <property name="text">
         <string>SubItem 21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SubItem 22</string>
        </property>
       </item>
      </item>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <row>
       <property name="text">
        <string>1</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/checkbox.png</normaloff>:/images/checkbox.png</iconset>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/ok_button.png</normaloff>:/images/ok_button.png</iconset>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Itens</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/radiobutton-off.png</normaloff>:/images/radiobutton-off.png</iconset>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valores</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/radiobutton.png</normaloff>:/images/radiobutton.png</iconset>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>11</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/textbold.png</normaloff>:/images/textbold.png</iconset>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>21</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/textcenter.png</normaloff>:/images/textcenter.png</iconset>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>22</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>lineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>tableWidget</receiver>
   <slot>selectRow(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
